hugegraph-core/src/main/java/com/baidu/hugegraph/traversal/optimize/HugeTraverser.java

```diff
@@ -137,6 +139,88 @@ public HugeTraverser(HugeGraph graph) {
         return paths;
     }
 
+    public List<Path> customizedPaths(List<HugeVertex> vertices,
```

Comment:
move to an independent class like [RankAlgorithm](https://github.com/hugegraph/hugegraph/pull/274/files)